games/nethack/Portfile

```diff
@@ -60,13 +65,18 @@ destroot.keepdirs  "${destroot}${prefix}/var/games/nethack/save/"
 
 post-destroot {
     if {${subport} eq "nethack"} {
-        reinplace "s|${destroot}||" "${destroot}${prefix}/bin/nethack"
-
         # Prevent existing preferences files being overwritten.
         foreach f { logfile record sysconf } {
             file rename ${destroot}${prefix}/var/games/nethack/${f} \
                 ${destroot}${prefix}/var/games/nethack/${f}.dist
         }
+        reinplace "s|${destroot}||" "${destroot}${prefix}/bin/nethack"
+    } else {
+        foreach f { logfile record } {
+            file rename ${destroot}${prefix}/share/nethackdir/${f} \
+                ${destroot}${prefix}/share/nethackdir/${f}.dist
+        }
+        reinplace "s|${destroot}||" "${destroot}${prefix}/bin/nethack343"
```

Comment:
So if you use the `${name}` variable, instead of `nethack343` above, you can remove a lot of duplication because that will work for the main port and sub-port.

Reply:
Done

Reply:
Oh yeah, ${subport}. Good catch.

Reply:
Proof I tested it, heh